packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -147,6 +157,7 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
                   ...metadata?.apps,
                   stripe: {
                     ...metadata?.apps?.stripe,
+                    enabled: false,
```

Comment:
adding it here to fix: lnstalling an app to a team enables it on all the team-events by default